sentry.gradle

```diff
@@ -164,8 +164,10 @@ def extractReleasesInfo() {
     android.applicationVariants.each { variant ->
 
         variant.outputs.each { output ->
-            def versionCode = output.getVersionCode()
-            def releaseName = "${variant.getApplicationId()}@${variant.getVersionName()}+${versionCode}"
+            def defaultVersionCode = output.getVersionCode()
```

Comment:
I'm a bit worried this change has no test to validate it.
Is it possible to add some tests even if just an integration test?

Reply:
I don't have experience enough to write tests for Gradle scripts

Reply:
Hmm we can take it from here ourselves then, it is next on our goals anyway to improve auto source map uploading by completely rewriting it and having it use the release and dist from your JS code without needing to pass an environment variable.

Reply:
The main problem that the sentry CLI will be invoked by Gradle and no way to modify \ change argument with what it was executed

Reply:
since release and dist can be set at runtime, but the Gradle script is executed at build time, the chances to get a release and dist misaligned are, even more, I'd say that we have to figure out how to allow custom dist and release to be set on JS but syncing it down to the native layers, but being sure that source maps are also being uploaded with these new values, the fix isn't as simple as this PR though, it'd require more changes and I'm unsure how to do it before a deeper investigation.

Reply:
I think synchronization source maps and setting values in the runtime is a developer's care. So it isn't a blocker from a point of view

This ability already implemented for iOS: https://github.com/getsentry/sentry-cli/pull/967

and I still waiting for these changes for the Android Gradle script

Reply:
if iOS also has the ability to do so, I'm fine with the changes, can you add an entry to changelog @retyui? thanks